src/features/defi/providers/idle/components/IdleManager/Deposit/components/Deposit.tsx

```diff
@@ -36,6 +36,7 @@ type DepositProps = StepComponentProps & {
   onAccountIdChange: AccountDropdownProps['onChange']
 } & StepComponentProps
 
+const idleInvestor = getIdleInvestor()
```

Comment:
just like `getChainAdapterManager()` - we can use this inside components directly, rather than instantiating outside components

my concern with doing this is w/ testing - the act of importing this file instantiates the investor - rather than it happening deeper down in the tree at runtime lazily

Reply:
Yeah my main concern was the *theoretical* unsafety of not being able to have `getIdleInvestor()` (the call) as a dependency, but in the case of this singleton, the *effective* underlying value will always be the same so we can just have `getIdleInvestor` as a dependency (the function, not the call)

Reply:
10ca1a1dde1b0afc06a96d993f3b0c8eb9a8d82b